packages/qdrant-loader/tests/unit/core/pipeline/workers/test_embedding_worker.py

```
@@ -418,3 +464,238 @@ async def empty_iterator():
+            embedding_service=self.mock_embedding_service,
+            max_workers=2,
+            queue_size=1000,
+            shutdown_event=self.mock_shutdown_event,
+        )
+        self.mock_embedding_service.batch_size = 1
+
+        hold = asyncio.Event()
+        started: list = []
+        cancelled: list = []
+
+        async def blocking_get_embeddings(contents):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `blocking_get_embeddings`

(ANN202)